src/PKSim.Presentation/DTO/Mappers/ExpressionParametersToSimulationExpressionsDTOMapper.cs

```diff
@@ -90,6 +90,7 @@ private ExpressionContainerDTO expressionContainerFor(IParameter relativeExpress
 
             group = _groupRepository.GroupByName(CoreConstants.Groups.VASCULAR_SYSTEM);
             expressionContainerDTO.ContainerPathDTO = _representationInfoRepository.InfoFor(RepresentationObjectType.CONTAINER, containerNameForGlobalExpression(relativeExpression.Name)).ToPathElement();
+            expressionContainerDTO.Sequence = relativeExpression.Sequence;
```

Comment:
In this particular case, I am, setting the sequence from the parameter. And won't update it after that.